Sources/HeroContext.swift

```diff
@@ -235,12 +245,18 @@ extension HeroContext {
       if !(view is UINavigationBar), let contentView = snapshot.subviews.get(0) {
         // the Snapshot's contentView must have hold the cornerRadius value,
         // since the snapshot might not have maskToBounds set
+        if #available(iOS 11, *) {
+          contentView.layer.maskedCorners = view.layer.maskedCorners
+        }
         contentView.layer.cornerRadius = view.layer.cornerRadius
         contentView.layer.masksToBounds = true
       }
 
-      snapshot.layer.allowsGroupOpacity = false
+      if #available(iOS 11, *) {
+        snapshot.layer.maskedCorners = view.layer.maskedCorners
+      }
       snapshot.layer.cornerRadius = view.layer.cornerRadius
+      snapshot.layer.allowsGroupOpacity = false
```

Comment:
Maybe this isn't needed,

> The default value is read from the boolean UIViewGroupOpacity property in the main bundle’s Info.plist file. If no value is found, the default value is true for apps linked against the iOS 7 SDK or later and false for apps linked against an earlier SDK.

https://developer.apple.com/documentation/quartzcore/calayer/1621277-allowsgroupopacity
```suggestion
```